app/validators/blocks/list_collector_validator.py

```diff
@@ -110,20 +110,24 @@ def validate_other_list_collectors(self):
                 )
 
     def validate_repeating_blocks_list_collectors(self, same_type, other_type):
-        if self.block.get("repeating_blocks"):
-            list_name = self.block["for_list"]
-
-            list_collectors_other_type = self.questionnaire_schema.get_other_blocks(
-                self.block["id"], for_list=list_name, type=other_type
-            )
-            other_list_collectors_same_type = (
-                self.questionnaire_schema.get_other_blocks(
-                    self.block["id"], for_list=list_name, type=same_type
-                )
+        if not self.block.get("repeating_blocks"):
+            return
+        list_name = self.block["for_list"]
+        section_id = self.questionnaire_schema.get_section_id_for_block(self.block)
+        list_collectors_other_type = self.questionnaire_schema.get_other_blocks(
```

Comment:
Since you're adding the results together, I think this can be simplified - remove the params `same_type` and `other_type` and then this could be 
`other_list_collectors = self.questionnaire_schema.get_other_blocks(self.block["id"], for_list=list_name)`
and use that in the if statement

Reply:
To clarify - this is checking that if you find any other list collectors in the section for the same list, thats an error because the section should only have 1?
Which sounds correct to me - just wanted to check

Reply:
It's removed now since we want to check for multiple list collector on section level, so I reverted it back to its original state when introduced by Kyle.